YAML/YAML.sublime-syntax

```diff
@@ -490,7 +490,9 @@ contexts:
     - match: \[
       scope: punctuation.definition.sequence.begin.yaml
       push:
-        - meta_scope: meta.flow-sequence.yaml
+        # Pushes an extra 'meta' on the scope stack that flow-pair can then clear,
+        # so we don't have to duplicate the context
+        - meta_scope: meta.sequence.flow.yaml meta.workaround.yaml
```

Comment:
`meta.workaround` is assigned to `[` and `]` and all other non-pair tokens, which might not be ideal.

Reply:
That's the downside of this workaround. I suppose `meta_content_scope` would excempt the brackets at least, but won't do anything for non-pair tokens, which is why I chose such a scope name.